core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -0,0 +1,131 @@
+<mat-card class="oppia-lesson-info-card">
+  <div class="oppia-lesson-info-content">
+    <p class="oppia-story-title">
+        {{ storyTitleTranslationKey | translate}}
```

Comment:
Hacky translation key is only used with a if statement, only if we have sufficient conditions met to display translation key. Maybe you can refer to this PR : https://github.com/oppia/oppia/pull/14882/files to see how it is being done.

Reply:
Will do.